### PR TITLE
Fix for the issue that LogManager.GetCurrentClassLogger returns only …

### PR DESCRIPTION
**Pull request by younghoandrewchaa**
Currently LogManager.GetCurrentClassLogger returns the full class name of CorelatorSharp, not the class that's using the logger. This fix will change the behaviour so that it correctly displays the calling class name

### CODE REVIEW

File: CorrelatorSharp.Logging/LogManager.cs [hunk 1]
```diff
@@ -14,5 +17,30 @@ public static ILogger GetLogger(string name)
...
+                declaringType = method.DeclaringType;
+                if (declaringType == null)
+                {
+                    className = method.Name;
+                    break;
+                }
+
+                framesToSkip++;
+                className = declaringType.FullName;
+            } while (declaringType.Module.Name.Equals("mscorlib.dll", StringComparison.OrdinalIgnoreCase));
```
Comment (ivanz) [inline CorrelatorSharp.Logging/LogManager.cs]: Can you look into a way to not make this dependent on 'mscorlib.dll', which doesn't exist in portable or .Net core? maybe you get do typeof(String).assembly/module or similar?

Comment (younghoandrewchaa) [inline CorrelatorSharp.Logging/LogManager.cs]: Sure. Let me look into it

**Comment by ivanz** [comment]
Can you add a test for this too please. Thanks in advance!

**Comment by younghoandrewchaa** [comment]
I've updated the pull request
- Added a unit test for LogManager
- Removed mscorlib.dll dependency

**Pull request closed without merging**